.github/workflows/linux.yml

```
@@ -415,7 +415,7 @@ jobs:
         uses: actions/cache@v1
         with:
           path: ${{ env.LibrariesPath }}/qt-cache
-          key: ${{ runner.OS }}-qt-${{ env.CACHE_KEY }}-${{ hashFiles('**/qtbase_5_12_8.diff') }}
+          key: ${{ runner.OS }}-qt-${{ env.CACHE_KEY }}-${{ hashFiles('**/qtbase_5_12_8.diff') }}-${{ hashFiles('**/qtbase_xkbfix_5_12_8.diff') }}-${{ hashFiles('**/qtwayland_5_12_8.diff') }}-${{ hashFiles('**/qtwayland_xkbfix_5_12_8.diff') }}
```

That's an insane line length, can it replaced with `**/qt*5_12_8.diff`?
```suggestion
          key: ${{ runner.OS }}-qt-${{ env.CACHE_KEY }}-${{ hashFiles('**/qt*5_12_8.diff') }}
```

---

fixed